Sources/AsyncDNSResolver/dnssd/DNSResolver_dnssd.swift

```diff
@@ -173,6 +180,12 @@ struct DNSSD {
 
         return try replyHandler.generateReply(records: records)
     }
+
+    private func deallocatePointers(serviceRefPointer: UnsafeMutablePointer<DNSServiceRef?>, replyHandlerPointer: UnsafeMutablePointer<DNSSD.QueryReplyHandler>) {
```

Comment:
Making this `static` would make the code a lot easier to follow.

Reply:
I made it static in the latest commit